src/router/index.js

```diff
@@ -14,6 +14,10 @@ const routes = [
         path: '/getting-started',
         component: () => import('@/views/HomePage.vue')
     },
+    {
+        path: '/design-principles',
```

Comment:
This'll work perfectly fine for a top-level page! However, if you prefer MDX, you could always create a directory `content/design-principles` with an `index.mdx` file in there and just write Markdown. That strategy would remove the need to write a custom Vue component, and also eliminate the need to add the router entry manually. Just a callout!